lib/client-options.js

```diff
@@ -89,6 +90,9 @@ function extend(baseOptions, userOptions) {
   if (!(options.policies.addressResolution instanceof policies.addressResolution.AddressTranslator)) {
     throw new TypeError('Address resolution policy must be an instance of AddressTranslator');
   }
+  if (!(options.policies.timestampGeneration instanceof policies.timestampGeneration.TimestampGenerator)) {
```

Comment:
I've added support for setting the policy to `null` (included in the jsdoc and unit test).